StarDb/Calibrations/tpc/TpcResponseSimulator.20171220.000001.C

```diff
@@ -0,0 +1,151 @@
+// $Id: TpcResponseSimulator.20171220.000001.C,v 1.2 2020/03/18 20:10:35 iraklic Exp $
+// $Log: TpcResponseSimulator.20171220.000001.C,v $
+// Revision 1.2  2020/03/18 20:10:35  iraklic
+// new Run17 params and Run18 params after separating sector20 from the rest
+//
+// Revision 1.3  2018/10/17 20:45:17  fisyak
+// Restore update for Run XVIII dE/dx calibration removed by Gene on 08/07/2018
+//
+// Revision 1.1  2018/08/06 15:41:44  fisyak
+// RunXVIII isobar dE/dx calibration
+//
+// Revision 1.1  2017/02/07 16:58:37  fisyak
+// Clean up
+//
+// Revision 1.1  2016/09/19 12:25:07  fisyak
+// Set/ReSet default values
+//
+// Revision 1.1  2014/01/27 14:35:11  fisyak
+// Add y2014 simulation (ideal) tables for y2014
+//
+// Revision 1.1  2014/01/09 23:15:10  fisyak
+// Run XIII, pp500p1
+//
+// Revision 1.3  2013/11/08 16:18:25  fisyak
+// T0 instruction
+//
+// Revision 1.2  2013/02/01 15:58:51  fisyak
+// Add handle for separate Inner and Outer sector time off set
+//
+// Revision 1.1  2012/04/03 14:06:55  fisyak
+// Speed up using  GetSaveL (__PAD_BLOCK__), sluggish shape histograms, Heed electron generation
+//
+// Revision 1.7  2012/04/03 14:06:55  fisyak
+// Speed up using  GetSaveL (__PAD_BLOCK__), sluggish shape histograms, Heed electron generation
+//
+// Revision 1.6  2012/01/18 13:57:57  fisyak
+// Adjust T0offset : Xianglei Zhu from Run 11 AuAu 27 & 19.6 GeV embedding
+//
+// Revision 1.5  2011/12/30 00:04:01  fisyak
+// Freeze parameters for y2011 base on TpcRS_2011_pp500LowLum_Q
+//
+// Revision 1.3  2011/10/11 19:09:23  fisyak
+// Add Yi Guo's tables for Run XI AuAu200 RFF dE/dx calibration
+//
+// Revision 1.2  2010/10/28 23:41:54  fisyak
+// extra t0 off set for Altro chip
+//
+// Revision 1.7  2010/06/14 23:36:08  fisyak
+// Freeze version V
+//
+// Revision 1.6  2010/05/24 21:39:53  fisyak
+// Fix bracket
+//
+// Revision 1.5  2010/05/24 16:07:20  fisyak
+// Add default dE/dx calibration tables, replace TpcAltroParameters and asic_thresholds_tpx by tpcAltroParams
+//
+// Revision 1.4  2010/04/19 15:05:58  fisyak
+// Final (2010_i) parameters for Run X
+//
+// Revision 1.3  2010/04/16 19:31:19  fisyak
+// Intermidiate version
+//
+// Revision 1.3  2010/04/04 23:14:33  fisyak
+// Add Row Correction
+//
+// Revision 1.2  2010/04/01 22:17:57  fisyak
+// Freeze version W
+//
```

Comment:
The history of changes is messed up and not relevant. 
Since it is a new file, suggest to remove

```suggestion
```